tools/wltests/platforms/akari_android-v4.9/build_images

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname $(realpath -s $0))
+BASE_DIR="$SCRIPT_DIR/../.."
+source "${BASE_DIR}/helpers"
+source "${PLATFORM_PATH}/definitions"
+
+################################################################################
+# Build all images required to test a new kernel
+################################################################################
+
+warn="\
+Update the 'definitions' file if needed, but be aware that the values must \
+be in sync with the userspace currently flashed on the device or it won't boot\
+"
+
+if [[ -z "$ANDROID_OS_VERSION" ]]; then
+    c_error "Could not read Android version from device. $warn"
+    exit $FATAL_ERROR
+elif [[ -z "$ANDROID_OS_PATCH_LEVEL" ]]; then
+    c_error "Could not read Android patch level from device. $warn"
+    exit $FATAL_ERROR
+fi
+
+warn="\
+They can be found at \
+https://android.googlesource.com/platform/prebuilts/misc/+/master/linux-x86/\
+"
+from_path=0
+
+for var in DTC_EXT DTC_OVERLAY_TEST_EXT; do
+    if [[ -z ${!var} ]]; then
+	echo "tt"
+	c_warning "$var not set, will attempt to set it by looking at the PATH"
+	from_path=1
+    fi
+done
+
+if [[ -z $from_path ]]; then
+    for tool in dtc ufdt_apply_overlay; do
+	if [[ -z $(which $tool) ]]; then
```

Comment:
Yeah, I just wanted to explicitly warn about "variable X not set, looking into your PATH" so that the user knows which variable to set and he/she can re-run wltest easily without having to look into the actual code. I can try to come up with a middle ground.